### DISCUSSION TITLE
CoolProp Module Documentation

### DISCUSSION THREAD

**Discussion by NCJazzMan65** [IDAES/idaes-pse#781] [General]
The latest 1.13 release of IDAES has CoolProp integration but I can't find any documentation on this new feature.  Will documentation be added to the online documentation soon?

**Comment by andrewlee94** [comment]
@NCJazzMan65 We do have documentation in the works, but unfortunately it has been pushed to a lower priority for the moment, and I cannot say for sure when we will b able to get back to it.

In the mean time, you can see the new interface being applied in the code tests, specifically line 244 onwards in https://github.com/IDAES/idaes-pse/blob/main/idaes/models/properties/core/coolprop/tests/test_coolprop_wrapper.py.
